### DISCUSSION TITLE
API v4 GraphQL Retrieve RepositoryInvitation?

### DISCUSSION THREAD

**Discussion by nmichlo** [community/community#24876] [Apps, API and Webhooks]
<p>I am probably missing something obvious but, I cannot seem to construct an equivalent GraphQL query to retrieve a list of users for a repository as seen in the version <a href="https://developer.github.com/v3/repos/invitations/#list-invitations-for-a-repository" rel="nofollow noopener">3 API</a>.</p>
<p>I have taken a look at the schema and can see the relevant type ‘<a href="https://developer.github.com/v4/object/repositoryinvitation/" rel="nofollow noopener">RepositoryInvitation</a>’ but I cannot seem to find the relevant fields that result in this information being retrieved from a query or specifically the query below.</p>
<pre><code>{
  viewer {
    organization(login: "&lt;name&gt;") {
      repositories(first: 5) {
        edges {
          node {
            collaborators(first: 5) {
              edges {
                node {
                  login
                }
              }
            }
            name
          }
        }
      }
      login
    }
    login
  }
}
</code></pre>

**Comment by lee-dohm** [comment, marked as answer]
<p>As of right now, there isn’t a method to retrieve the information you’re asking for via the v4 API. You’ll need to continue to use the v3 API for now.</p>
<p>I hope that helps!</p>

**Comment by nmichlo** [comment]
<p>No problem, thank you for clarifying.</p>
<p>The existence of the data type had me confused.</p>
